amp/WEB-INF/src/org/digijava/module/aim/annotations/activityversioning/ActivityVersioningService.java

```diff
@@ -0,0 +1,721 @@
+package org.digijava.module.aim.annotations.activityversioning;
```

Comment:
since we already have an ActivityVersioningUtil class it would be better to place the code there since we already have related functions in the same class. We would need a ticket tin order to refactor that class bases on common methods in order to make it smaller

Reply:
still in a service and not in util.